backend/utilities/email.js

```diff
@@ -0,0 +1,60 @@
+import nodemailer from 'nodemailer';
+
+const email = process.env.EMAIL;
+const emailPassword = process.env.EMAIL_PWD;
+
+// TO DO change url for production
+const url = process.env.FRONTEND_URL_DEV;
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: email,
+    pass: emailPassword,
+  },
+});
+
+const getRootUrl = () => {
+  if (url) return url;
+  return url;
+  // return req.protocol + '://' + req.get('host');
+};
+
+const sendMail = (recipient, subject, content) => {
+  const mailOptions = {
+    from: 'hypertube@no-reply.com',
+    to: recipient,
+    subject,
+    html: content,
+  };
+
+  transporter.sendMail(mailOptions);
+};
+
+const sendResetEmail = (recipient, name, token, req) => {
+  const link = `${getRootUrl(req)}/recoverylink?token=${token}`;
```

Comment:
For me this resulted in:
x-webdoc://3F684866-C1BB-4CDB-82BF-90D78ADDBD65/undefined/recoverylink?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYwMzc4MzI0ZWMzZTk0MDAxZTdlNzI4OSIsImxhbmciOiJkZSIsImlhdCI6MTYxNDI1MTUwMn0.IHq_v-QZm-5Q8_N1fRK8WWbDzfo88Pu166cEzMCYxnI

Reply:
I couldn't replicate this error, I get a correct link on my side.
Have you added FRONTEND_URL_DEV to .env and ran `make build`?

Reply:
That was it, the url was missing. Everything works now, so I will accept the merge